web/cobrands/northamptonshire/base.scss

```diff
@@ -15,13 +15,27 @@
     width: 190px;
     height: 41px;
     padding: 11px 0;
```

Comment:
If we’re changing stuff about this logo, why keep this odd padding rule? Surely just a width and height is required, no need for padding if we’re using just the background-image?

Reply:
I’d be tempted to remove both the `padding` and `height` here, and leave it to inherit the `height: 60px` from the core styles. It’s only 3px off the 63px you’re currently using, and it’d mean less custom code :-)